sound/soc/sof/loader.c

```diff
@@ -617,6 +618,9 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 	/* fw boot is complete. Update the active cores mask */
 	sdev->enabled_cores_mask = init_core_mask;
 
+	/* increment reset count */
+	atomic_add(1, &sdev->reset_count);
```

Comment:
You don't see to be using the reset_count field for anything, so what is it needed for again?

Reply:
@plbossart look at the last patch:
```
+	unsigned int reset_count = atomic_read(&sdev->reset_count);
...
+		resp->reset_status = reset_count == client->reset_count ?
+			SOF_VIRTIO_IPC_RESET_NONE : SOF_VIRTIO_IPC_RESET_DONE;
```

Reply:
it's impossible to review with GitHub

Reply:
> it's impossible to review with GitHub

@plbossart I've been saying that since more or less day 1...